runtime/src/lib.rs

```diff
@@ -1017,6 +1019,40 @@ impl ismp_grandpa::Config for Runtime {
     type WeightInfo = weights::ismp_grandpa::ZKVWeight<Runtime>;
 }
 
+impl pallet_hyperbridge::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type IsmpHost = Ismp;
+}
+
+impl pallet_assets_mock::Config for Runtime {
+    type Currency = Balances;
+    type AssetId = u32;
+    type Balance = Balance;
+}
+
+parameter_types! {
+    pub const Decimals: u8 = 18;
+    pub const NativeAssetId: u32 = 0;
+}
+
+parameter_types! {
+    /// Account of the treasury pallet.
+    pub AssetAdmin: AccountId = Treasury::account_id();
+}
+
+impl pallet_token_gateway::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type Dispatcher = Ismp;
+    type Assets = pallet_assets_mock::Pallet<Runtime>;
+    type NativeCurrency = Balances;
+    type AssetAdmin = AssetAdmin;
+    type CreateOrigin = EnsureRoot<AccountId>;
```

Comment:
It seems that this breaks the automatic benchmarks for us; given that these benchmarks do not use any `try_successful_origin()`, I would suggest using a different `CreateOrigin` when compiling with the feature `runtime-benchmarks`?